src/app/submissions/submissions.service.ts

```diff
@@ -671,6 +695,38 @@ export class SubmissionsService {
     };
   }
 
+  aggregateRatingScaleResponses(question, submissions) {
```

Comment:
Is there a reason we can't use `aggregateQuestionResponses` instead? There's a bit of duplication here that I think we can avoid.

This function could call that function and then do some data manipulation/sorting if needed.

Reply:
Just combined the `aggregateRatingScaleResponses` with the `aggregateQuestionResponses`